pages/index.js

```diff
@@ -40,17 +40,14 @@ function Index({ trackId, backSideContent, latestId }) {
   }, [enterKey]);
 
   React.useEffect(() => {
-    if (localStorage.getItem("refresh") === "true"){
-      localStorage.setItem("refresh", false)
+    if (sessionStorage.getItem("refresh") === "true"){
```

Comment:
Shouldn't we set `refresh` to `false` after this line?

```js
sessionStorage.setItem("refresh", false);
```

Reply:
No. Just set once per session, without `onbeforeunload`, and show refresh-messages always except the first loading.